docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+
+# NOTE: You must set WOMBAT_TAG, RHINO_TAG and PLOS_THEMES_REPO environment variables
+#       as well as all environment variables not explicitly defined below (e.g. CAS_URL)
+
+services:
+
+  wombat:
+    image: plos/wombat:${WOMBAT_TAG}
```

Comment:
since docker-compose will only be used for local dev (right?) what do you think about just using `latest` here? (and below for rhino)

Reply:
fair point and removes the need for that var. updated.